site/gatsby-site/cypress/integration/downloadIndex.js

```diff
@@ -0,0 +1,92 @@
+import { gql } from '@apollo/client';
+import path from 'path';
+
+describe('Download Algolia Index', () => {
+  const url = '/downloadIndex';
+
+  const downloadsFolder = Cypress.config('downloadsFolder');
+
+  it('Should return a properly constructed Algolia index', () => {
+    cy.visit(url);
+
+    cy.query({
+      query: gql`
+        query {
+          reports(limit: 9999) {
```

Comment:
It may be better to set this low for tests so the test will run faster.

Reply:
Tested just in case, but it doesn't seem to make that much of a difference, maybe once there are more reports.